data/172/939/588/5/1729395885.geojson

```diff
@@ -5,11 +5,17 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.014388,
-    "geom:area_square_m":139996020.496609,
+    "geom:area_square_m":139996020.496601,
     "geom:bbox":"-95.188061,38.037702,-95.076501,38.168748",
     "geom:latitude":38.1031,
     "geom:longitude":-95.132141,
     "iso:country":"US",
+    "label:eng_x_preferred_placetype":[
+        "township"
+    ],
+    "label:fra_x_preferred_placetype":[
```

Comment:
Nice

Reply:
I've tried to set FRA labels for the placetypes in all countries (so they have ENG and FRA at minimum).

In some countries like Estonia and USA not all the places at a given placetype have the same type. Looking at France that is kinda also true... but they are all some variation on "commune" there, so I've gone with generic "commune" here as a 1st pass.